.eslintrc

```diff
@@ -25,6 +25,17 @@
       "rules": {
         "@typescript-eslint/no-var-requires": "off"
       }
+    },
+    {
+      "files": ["*.js", "*.jsx", ".ts", ".tsx"],
+      "processor": "@graphql-eslint/graphql"
+    },
+    {
+      "files": ["*.gql", "*.graphql"],
+      "extends": [
+        "plugin:@graphql-eslint/schema-recommended",
+        "plugin:@graphql-eslint/operations-recommended"
+      ]
     }
```

Comment:
Ah, the linter is in state right now as we move shared config things into `packages/eslint-config`.  This top-level .eslintrc won't be around much longer! 

Can you copy this logic (and required plugin dependencies) to `packages/eslint-config` for now?